Utils/azure-toolkit-ide-libs/azure-toolkit-ide-vm-lib/src/main/java/com/microsoft/azure/toolkit/ide/vm/VirtualMachineActionsContributor.java

```diff
@@ -36,6 +37,11 @@ public void registerActions(AzureActionManager am) {
             .enabled(s -> s instanceof VirtualMachine && ((VirtualMachine) s).getFormalStatus().isRunning());
         am.registerAction(ADD_SSH_CONFIG, new Action<>(ADD_SSH_CONFIG, addSshConfigView));
 
+        final ActionView.Builder connectSshView = new ActionView.Builder("Connect(ssh)", AzureIcons.Action.RESTART.getIconPath())
```

Comment:
let's rename the action to `Connect Using SSH`.
refer to https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/factoryos/connect-using-ssh?view=windows-11.